modules/admin_manual/pages/configuration/general_topics/impersonate_users.adoc

```diff
@@ -29,6 +29,28 @@ Anything that you see until you log out will be what that user would see.
 
 When you're ready to stop impersonating the user, log out and you will return to your normal user session.
 
+== Define Who can Configure Impersonate
```

Comment:
```suggestion
== Define Who can use Impersonate
```

Actually this defines the users who have the Impersonate app "enabled" for them. (all users, or just the users in these groups)

For a user who is not an admin or sub-admin, they can be impersonated (but cannot impersonate anyone).

An admin can impersonate any of the users in the groups.

A sub-admin can impersonate users in their group(s) (if those extra settings are enabled).

Only admins can "configure Impersonate" (i.e. set the various system-wide settings for the Impersonate app)

Reply:
I do not get it...
Where is the difference in between:
`this defines the users who have the Impersonate app "enabled" for them`
`A sub-admin can impersonate users in their group(s)`
because in both settigs you can define groups (and only groups), but the first one is at the app section and the second one is defined at the user section at the 3 radiobuttons.
I have tested this and if you add a group in one part it does not show up at the other one

Reply:
They are 2 completely separate things:
1) gives "access" to the app for whole groups of users (the potential to do impersonating or be impersonated)
then step 2) give rights to sub-admins to do impersonating